shell/common/shell.cc

```diff
@@ -1200,6 +1200,11 @@ const Settings& Shell::OnPlatformViewGetSettings() const {
   return settings_;
 }
 
+// |PlatformView::Delegate|
+void Shell::UpdateImpellerState(bool impeller_enabled) {
+  settings_.enable_impeller = impeller_enabled;
```

Comment:
ta-da

Reply:
basically, after we create the android context - determine if we fell back to Skia, then update the (formerly const) settings object so the existing engine code keeps working.